public/regdesk.html

```diff
@@ -4,10 +4,28 @@
 <head>
   <meta charset="UTF-8">
   <title>FC RegDesk</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="color-scheme" content="light dark">
+  <meta name="theme-color" content="#111111" media="(prefers-color-scheme: light)">
+  <meta name="theme-color" content="#eeeeee" media="(prefers-color-scheme: dark)">
+  <link rel="stylesheet" href="regdesk.css" />
 </head>
 
 <body>
-    <h1>This will be an awesome page with stuff!</h1>
+  <div class="container-fluid">
+      <div class="row" style="padding-top: 16px;">
```

Comment:
Inline styles are considered a poor practice[1], and specifying pixels is also fallen out of fashion[2]. 

[1] https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css
[2] https://stackoverflow.com/a/43131958/1204809


This is all temporary code, so I'm not gunna block on it by any means. But figured you might be interested in learning that.

Reply:
I was vaguely aware that it was frowned upon but not the deeper reasons, that was useful, thanks!